src/main/java/com/minecolonies/entity/ai/citizen/fisherman/EntityAIWorkFisherman.java

```diff
@@ -212,6 +203,12 @@ private AIState prepareForFishing()
     {
         if (checkOrRequestItems(new ItemStack(Items.FISHING_ROD)))
         {
+            if(worker != null)
```

Comment:
I'd put this all in a need fishing rod sound method. So the private method does the null check and plays the sound.